understanding/22/target-size-minimum.html

```diff
@@ -46,11 +47,12 @@ <h2>Intent of Target Size (Minimum)</h2>
 
         <p>This Success Criterion defines a <em>minimum</em> size; using larger sizes will help many people use targets more easily. Meeting <a href="../21/target-size-enhanced.html">2.5.5 Target Size (Enhanced)</a> is recommended whenever possible.</p>
 
-        <p>The requirement is for targets to be at least 24 by 24 CSS pixels in size. There are three exceptions:</p>
+        <p>The requirement is for targets to be at least 24 by 24 CSS pixels in size. There are four exceptions:</p>
 
         <ol>
             <li>If a target is smaller than 24 by 24 CSS pixels but has sufficient spacing, it passes. The spacing is sufficient if the offset between targets is at least 24 CSS pixels. Target offset measures the distance between targets, measured from the farthest point of one target to the nearest point of the adjacent target, and is assessed for each of the adjacent targets. What edge of the target is <em>farthest</em> depends on the relative position of the neighboring target. For example, if considering distance above the target, the farthest edge would be the bottom of the target, and the offset would be measured from this bottom edge to the bottom edge of the neighboring target above. Note that where the targets differ in size or shape, the offset between targets may also differ. The offset distance must be at least 24 CSS pixels in all cases to fall under this exception.</li>
             <li>The Success Criterion does not apply to inline targets in blocks of text – for example, text links inside a sentence or paragraph. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of texts in smaller text sizes, maintaining a 24 CSS pixels offset between links in adjacent lines of text would require a large line height which can be undesirable in many design contexts. Also, inline numeric footnotes common in scientific texts may sometimes have a width well below 24 CSS pixels. Note: Stacked lists of links, as in navigation structures, do not count as inline links. Authors can anticipate the relative position of these links and accommodate sufficient target spacing.</li>
+            <li>Some native elements like `select` are styled by user-agents. Another example is `input type="number"` that in some user-agents brings up small arrows to increase or decrease the value. If authors cannot style a native control so that its size is determined by the user agent, this control is exempt.</li>
```

Comment:
```suggestion
            <li>Some native elements like <code>select</code> are styled by user-agents. Another example is <code>input type="number"</code> that in some user-agents brings up small arrows to increase or decrease the value. If authors cannot style a native control so that its size is determined by the user agent, this control is exempt.</li>
```